mreg/models.py

```diff
@@ -718,10 +718,22 @@ def __str__(self):
         return "%s" % self.name
 
 
+class Label(BaseModel):
+    name = LCICharField(max_length=64, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
Add restrictions here via `validators`.

Reply:
Added a validator to check for spaces in a string, and used the `blank` option to disallow empty values for the description:
https://github.com/unioslo/mreg/blob/17eb02178d5f4025b27754ab3ce4def75e06fd8e/mreg/models.py#L722-L724

https://github.com/unioslo/mreg/blob/17eb02178d5f4025b27754ab3ce4def75e06fd8e/mreg/validators.py#L172-L174